plugins/header_rewrite/parser.cc

```diff
@@ -244,6 +244,14 @@ Parser::cond_is_hook(TSHttpHookID &hook) const
     hook = TS_REMAP_PSEUDO_HOOK;
     return true;
   }
+  if ("TXN_START_HOOK" == _op) {
```

Comment:
I don't remember how we deal with things like this, but this hook would not be viable when invoked as a remap plugin. TXN_START happens before that; meaning, this can only be used when running header_rewrite as a global plugin.

Reply:
If TXN_START is invoked in remap, the header value is not added.

Reply:
Yeh, but I'm thinking it should maybe be a warning or even an error if TXN_START_HOOK is used in a remap plugin.

Reply:
I believe a warning would suffice.